### PR TITLE
Adcs refactor

### PR DESCRIPTION
**Pull request by ppapadeas**

### CODE REVIEW

**Comment by ppapadeas** [comment]
@nchronas please review.

File: satnogsclient/web/static/js/upsat-control.js [hunk 1]
```diff
@@ -338,23 +362,48 @@ $(document).ready(function() {
...
             // TODO: Is app_id needed in time service?
             //app_id = $('#service-param-time-app_id').val();
-            app_id = 7;
-            type = 0;
-            ack = 0;
+            app_id = 3;
+            type = 1;
+            ack = $('#service-param-adcs-ack').val();
 
             service_type = 3;
```
Comment (nchronas) [inline satnogsclient/web/static/js/upsat-control.js]: TC_FUNCTION_MANAGEMENT_SERVICE is 8 not 3

File: satnogsclient/web/static/js/upsat-control.js [hunk 2]
```diff
@@ -338,23 +362,48 @@ $(document).ready(function() {
...
             service_subtype = 23;
```
Comment (nchronas) [inline satnogsclient/web/static/js/upsat-control.js]: default subtype is 1

File: satnogsclient/web/static/js/upsat-control.js [hunk 3]
```diff
@@ -338,23 +362,48 @@ $(document).ready(function() {
...
+            dest_id = $('#service-param-adcs-dest_id').val();
 
             data = [];
-            ascii_to_dec($('#service-param-service-tle').val().split(''), data);
-            data.unshift(6);
-            //number of TLE chanacters
-            if (data.length != 137) {
-                alert("TLE shouldnt be: " + data.length);
-                return 0;
+            data.splice(0, 0, service_type);
```
Comment (nchronas) [inline satnogsclient/web/static/js/upsat-control.js]: data[0] = 3; //function id, set value
data.splice(0, 0, 3);

Comment (ppapadeas) [inline satnogsclient/web/static/js/upsat-control.js]: It is 3 as service_type = 3 on line 369

File: satnogsclient/web/static/js/upsat-control.js [hunk 4]
```diff
@@ -338,23 +362,48 @@ $(document).ready(function() {
...
+                var g1 = $('#service-param-service-g1').val();
+                var g2 = $('#service-param-service-g2').val();
+                var g3 = $('#service-param-service-g3').val();
+                data.splice(1, 0, 15);
+                data.splice(2, 0, g1);
+                data.splice(4, 0, g2);
+                data.splice(6, 0, g3);
+            } else if (adcs_action == "ADCS_TLE") {
+                ascii_to_dec($('#service-param-service-tle').val(), data);
+                data.unshift(service_type);
```
Comment (nchronas) [inline satnogsclient/web/static/js/upsat-control.js]: why are you using unshift?
data[0] should be 3.

File: satnogsclient/web/static/js/upsat-control.js (hunk 3 above)
Comment (nchronas) [inline satnogsclient/web/static/js/upsat-control.js]: ok, as service type can be confusing with the ecss packet service type, can you change it with something like function_type?

**Pull request merged**